doc/server_API.md

```diff
@@ -230,8 +216,8 @@ On success:
 
 ```json
 {
-  "whitelist_id":12345,
-  "module_id":"125116623"
+  "whitelist_unique_id":12345,
```

Comment:
Once again, i think simple `whitelist_name` would suffice, as it better reflects database structure, but this is only my opinion.

If you decide to fix it, also fix it in `unbind_whitelist_from_module`